examples/plotting/solar_cycle_example.py

```diff
@@ -13,14 +14,18 @@
 from sunpy.time import parse_time
 
 ###############################################################################
-# For this example we will use the SunPy sample data. This code snippet grabs
-# the most current NOAA solar cycle data as a ``TimeSeries``.
+# For this example we will use sunpy's sample data which has an old
+# sunspot record and prediction from (SWPC) part of the National Oceanic
```

Comment:
Thanks for this PR! Can you clarify what you mean by "an old sunspot record"? This sounds somewhat ambiguous. Is this referencing an outdated sunspot record or a record of an older solar cycle? If it is the latter, is it the most recent past cycle (Solar Cycle 24) or some other cycle or a collection of multiple cycles?

Reply:
It is old in the sense the data it uses is from the start of 2016.

Reply:
Okay, then this should be clarified. We should explicitly mention what time range the data spans.

Reply:
The file `NOAAINDICES_TIMESERIES` has data from 1991 until 2015. I might advocate that we remove it from sample data and just download it when needed since it will always be out of date. `NOAAPREDICT_TIMESERIES` goes from 2015-07 to 2019-12.

Reply:
Clarified the meaning of 'old' data